hack/tool/backup_restore/cmd/restore.go

```diff
@@ -109,3 +110,21 @@ func restore(ctx context.Context) error {
 
 	return app.WaitConfiguration(ctx, resourceOwner.GetConfigurationNamespacedName())
 }
+
+// presetTFBackendNS try to set the "TERRAFORM_BACKEND_NAMESPACE" environment variable
+func presetTFBackendNS() {
+	backendNS := os.Getenv(app.TFBackendNS)
+	if backendNS != "" {
+		goto end
+	}
```

Comment:
The logic isn't that complex. Using `If Else` is enough

Reply:
OK